omod/src/test/java/org/openmrs/module/metadatasharing/PublishTest.java

```diff
@@ -17,8 +17,6 @@
 import java.lang.reflect.Field;
 import java.lang.reflect.Modifier;
 
-import junit.framework.Assert;
```

Comment:
Could you help me to understand why these changes were made to this file? Do they need to be in this PR?